test/IDE/complete_at_top_level.swift

```diff
@@ -10,7 +10,8 @@
 
 // RUN: %target-swift-ide-test -code-completion -source-filename %s -code-completion-token=TOP_LEVEL_VAR_INIT_1 > %t.toplevel.txt
 // RUN: %FileCheck %s -check-prefix=TOP_LEVEL_VAR_INIT_1 < %t.toplevel.txt
-// RUN: %FileCheck %s -check-prefix=TOP_LEVEL_VAR_INIT_1_NEGATIVE < %t.toplevel.txt
+// FIXME: rdar://problem/56755598
+// RUN: not %FileCheck %s -check-prefix=TOP_LEVEL_VAR_INIT_1_NEGATIVE < %t.toplevel.txt
```

Comment:
The test case is 
```
var topLevelVar1 = #^TOP_LEVEL_VAR_INIT_1^#
// Check that the variable itself does not show up.
// TOP_LEVEL_VAR_INIT_1_NEGATIVE-NOT: topLevelVar1
```
This used to work only because typechecking this decl was failed. Now that it's typechecked to `UnresolvedType` so it revealed the existing issue(rdar://problem/56755598).